src/python/pants/engine/legacy/round_engine.py

```diff
@@ -2,64 +2,45 @@
 # Licensed under the Apache License, Version 2.0 (see LICENSE).
```

Comment:
I merged `engine.py` and `round_manager.py` into the same file `round_engine.py`. Those two were only being used by `round_engine.py`. No changes made to any of the code - only moving things around.